src/internal/components/chart-plot/focus-outline.tsx

```diff
@@ -1,20 +1,46 @@
 // Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
 // SPDX-License-Identifier: Apache-2.0
-import React, { useEffect, useRef } from 'react';
+import React, { useEffect, useRef, useState } from 'react';
 
 import styles from './styles.css.js';
-import useFocusVisible from '../../hooks/focus-visible/index';
 import { Offset } from '../interfaces';
+import { isModifierKey } from '../../hooks/focus-visible';
```

Comment:
Would it make sense to move the util to `internal/keycode`?

Reply:
That is a very strange looking file (some copy of test-utils code for non-test context). Let's not move it for now, can change later if it becomes used anywhere outside focus-visible context